archicad_updates/ARCHICADUpdatesProcessor.py

```diff
@@ -85,34 +86,77 @@ def main(self):
         # Grab the available downloads.
         # Fixed URL (jutonium)
         response = self.download(
-            "https://graphisoft.com/ww/service/downloads/archicad-updates",
-            headers={"Accept": "application/json"},
+            "https://graphisoft.com/de/service-support/downloads"
         )
-        json_data = json.loads(response)
+
+        # Parse the response to get the json data.
+        json_response = re.search(r":categories='(.*)'", response.decode('utf-8')).group(1)
+        json_data = json.loads(json_response)
+
+        # Parse through the available categories for to identify the category id
```

Comment:
Suggestion: Parse through the available categories to identify the category id
(no "for" needed)